web/static/js/demographics-form.js

```diff
@@ -22,15 +22,20 @@ $gender.on('change', function () {
 });
 $gender.change();
 
-// Show state field if US is selected in the country field.
+// Show state and race/ethnicity fields if US is selected in the country field.
 const $country = $('#id_country');
 $country.on('change', function () {
-    // Show/hide state field based on USA or not
+    // Show/hide state and race/ethnicity fields based on USA or not
     if ($(this)[0].value === 'US') {
         showField('state');
+        showField('race_ethnicity_identification');
     } else {
         hideField('state');
         $(`#id_state`)[0].value = '';
+        hideField('race_ethnicity_identification');
+        $(`input[name="race_ethnicity_identification"`).each(function() {
+            this.checked = false;
+        });
```

Comment:
Very nice.  💯

Reply:
After looking more into this, there's more that needs to done or discussed.  The template will remove these fields when the a non-US country is selected.  This means that if you save the form with a non-US country and attempt to change to US the fields can't be show.  They're not there.  

The reverse is true.  If you save the form as US, you can toggle the fields.  They exist.  

Do we want to have the field toggle-able (maybe not a word ;) )? Or we want the fields not exist when a the rest of the world sees this form?

Reply:
>What about referring to/thinking of this field as "usa_race..." and setting
>the initial value to 'NA'/Not applicable for all forms outside the US?  So
>if someone started on the USA and toggled, the (actual) value given would
>remain. If someone started elsewhere the values would be set to NA on first
>save. If they then toggled to US, they would see a set of non-checked boxes
>for that question, and could update if desired.  Does that make sense?

@mekline Feel free to make comments in these conversation boxes.  It makes life easier when needing to context switch.  

I think you're suggesting we clear the selection when country is toggled from non-US to US.  Great suggestion.  We're currently doing that to some degree.  I think we can formally implement it.  

I am inferring you'd prefer we toggle the fields and clear them when shown?